src/app/components/split-view/index.html

```diff
@@ -0,0 +1,229 @@
+          </sky-docs-visual-example-text>
+          <sky-docs-visual-example-image>
+            <img src="~/assets/img/guidelines/split-view/option-list-type-grid.png" />
+            <div class="caption caption-warning sky-text-warning">Use caution when using grids as the list type as they do not perform well at narrow widths.</div>
+          </sky-docs-visual-example-image>
+        </sky-docs-visual-example>
+      <h3>List manipulation</h3>
+        <sky-docs-visual-example>
+          <sky-docs-visual-example-text>
+            <p>You may include a list toolbar so that users can filter, sort and search the list, or add new items to the list.</p>
+          </sky-docs-visual-example-text>
+          <sky-docs-visual-example-image>
+            <img src="~/assets/img/guidelines/split-view/option-toolbar.png" />
+          </sky-docs-visual-example-image>
+        </sky-docs-visual-example>
+    </sky-demo-page-content>
+
+    <sky-demo-page-content sectionHeading="Behavior and states">
+      <h3>Automatically advance to the next item</h3>
+        <div class="sky-text-block">
+          <p>If the user is likely to work through the list of items sequentially, automatically advance to the next item in the list when the user invokes the primary action on the record.</p>
+        </div>
+      <h3>Set focus in the workspace</h3>
+        <sky-docs-visual-example>
+          <sky-docs-visual-example-text>
+            <p>When the user selects a new item in the list, set focus on the first focusable element in the workspace so the user can immediately start working.</p>
+          </sky-docs-visual-example-text>
+          <sky-docs-visual-example-image>
+            <img src="~/assets/img/guidelines/split-view/behavior-focus.png" />
+          </sky-docs-visual-example-image>
+      </sky-docs-visual-example>
+      <h3>Change panel width</h3>
+        <sky-docs-visual-example>
+          <sky-docs-visual-example-text>
+            <p>The user can change the width of the panels by drag-and-drop or keyboard interactions. If the navigation list content is wider than the panel, the panel will have a horizontal scrollbar.</p>
+          </sky-docs-visual-example-text>
+          <sky-docs-visual-example-image>
+            <img src="~/assets/img/guidelines/split-view/behavior-resize.png" />
+          </sky-docs-visual-example-image>
+      </sky-docs-visual-example>
+    </sky-demo-page-content>
+    <h3>Responsiveness</h3>
+        <sky-docs-visual-example>
+          <sky-docs-visual-example-text>
+            <p>In smaller viewports the split view automatically switches from a side-by-side view to a panel-switching interaction where the user flips between the list and the workspace. The responsive workspace header should have iterator buttons so the user can move between items without having to go back to the list.</p>
```

Comment:
Add a comma after viewports?

Reply:
I created a branch to make some edits; I'll make this edit in my branch.